assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -124,7 +124,7 @@ function DashboardSummaryWidget( { Widget, WidgetReportZero, WidgetReportError }
 		);
 	}
 
-	if ( isZeroReport( today ) && isZeroReport( period ) && isZeroReport( daily ) ) {
+	if ( isZeroReport( period ) && isZeroReport( daily ) ) {
```

Comment:
We need to include the previous period in this check as well since it should only show the report zero component when both current and previous periods are zero data, not just the current. It looks like this was missing before but would be good to fix now.